src/pages/policies-and-audits/audits/[...slug].astro

```diff
@@ -0,0 +1,34 @@
+---
+import { getCollection } from 'astro:content';
+import PageLayout from '../../../layouts/PageLayout.astro';
+import SideNav from '../../../components/SinglePageSideNav.astro'
+import TopNav from '../../../components/TopNav.astro'
+export async function getStaticPaths() {
+  const entries = await getCollection('audits');
+  return entries.map(entry => {
+    const slug = entry.slug == './' ? undefined : entry.slug
+    return {
+      params: { slug: slug},
+      props: { entry, entries}
+    }
+  })
+}
+const { entry, entries} = Astro.props;
+const { Content, headings } = await entry.render();
+
+---
+
+<PageLayout title ={entry.data.title} >
+
+  <TopNav parent_nav_path="policies-and-audits/audits" parent_nav_text = "Audit Repository" />
```

Comment:
I wonder if the `TopNav` should be part of the menu component. There are two reasons:

1. It is semantically part of the navigation so one could argue it belongs inside the `<nav></nav>` If we want to change this, we will probably need to alter the `SideNav` component to accept an optional text/link. 
2. In the prototype this link is vertically aligned with the content's main heading. When this component is in its own div, it pushed the main content down so it is no longer at the top of the page.

Reply:
Thank you! Moved TopNav into Sidenav. code checked in